docs/src/running-validator/vote-accounts.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Vote Account Management
+---
+
+This page describes how to set up an on-chain _vote account_.   Creating a vote
+account is needed if you plan to run a validator node on Solana.
+
+All settings for a vote account can be configured at the time the account is
```

Comment:
Not sure I grok this sentence. It sounds like it could be reduced to to "The vote account can be configured when first created or after the validator is running."  Also, should there be some details on how to do that or what the configuration options are?